@javiercn 

```
  CSC : error CS5001: Program does not contain a static 'Main' method suitable for an entry point [D:\a\_work\1\s\src\Controls\samples\Controls.Sample.Sandbox\Maui.Controls.Sample.Sandbox.csproj]
         D:\a\_work\1\s\src\Controls\samples\Controls.Sample.Sandbox\Platforms\Windows\App.xaml.cs(10,4): error CS0103: The name 'InitializeComponent' does not exist in the current context [D:\a\_work\1\s\src\Controls\samples\Controls.Sample.Sandbox\Maui.Controls.Sample.Sandbox.csproj]

```

--- a/src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets
+++ b/src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets
@@ -47,6 +47,14 @@
       <!-- Remove the items in AppDesignerFolder ("Properties", by default) so they don't get copied to the output directory -->
       <Content Remove="$(AppDesignerFolder)\**" Condition="'$(AppDesignerFolder)' != ''" />
     </ItemGroup>
+
+  </Target>
+
+  <Target Name="_RemoveStaticWebAssetsDevelopmentManifest" BeforeTargets="GetCopyToOutputDirectoryItems">
+    <!-- Remove the development manifest from the output since its not used and it will otherwise get incorrectly loaded in production. -->
+    <ItemGroup>
+      <ContentWithTargetPath Remove="$(StaticWebAssetDevelopmentManifestPath)" />
+    </ItemGroup>
   </Target>
 
   <!-- Targets temporarily remove Content items in various folders so that they don't conflict with iOS/MacCatalyst SDK tasks -->

